interfaces/builtin/desktop.go

```diff
@@ -250,8 +259,15 @@ func (iface *desktopInterface) AppArmorConnectedPlug(spec *apparmor.Specificatio
 }
 
 func (iface *desktopInterface) MountConnectedPlug(spec *mount.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
+	appId := "snap." + plug.Snap().Name()
```

Comment:
I think this should be also limited to classic. On core there's no session yet. Not a blocker though.

Reply:
Oh, that's good then. +1